src/api/modelTypes.js

```diff
@@ -145,49 +108,155 @@ export type User = $ReadOnly<{|
    */
   avatar_url: AvatarURL,
 
-  // These properties appear in data from the server, but we ignore
-  // them. If we add these, we should try to avoid `avatar_url`
-  // falling out of sync with them.
-  // avatar_source: mixed,
-  // avatar_url_medium: mixed,
-  // avatar_version: mixed,
-
-  // profile_data added in commit 02b845336 (in 1.8.0);
-  // see also e3aed0f7b (in 2.0.0)
-  // (This one doesn't appear in `/users` responses.)
-  profile_data?: empty, // When we need this, describe its actual type.
+  // If we use this, avoid `avatar_url` falling out of sync with it.
+  -avatar_version: number,
+
+  full_name: string,
+  is_admin: boolean,
+
+  // TODO(server-3.0): New in FL 8
+  is_owner?: boolean,
+
+  // TODO(server-5.0): New in FL 73
+  is_billing_admin?: boolean,
+
+  // TODO(server-4.0): New in FL 59
+  // Possible values are:
+  //   Organization owner => 100
+  //   Organization administrator => 200
+  //   Organization moderator => 300
+  //   Member => 400
+  //   Guest => 600
+  role?: number,
+
+  // `null` if the user isn't a bot.
+  // `1` for a `Generic` bot.
+  // `2` for an `Incoming webhook` bot.
+  // `3` for an `Outgoing webhook` bot.
+  // `4` for an `Embedded` bot.
+  bot_type: number | null,
+
+  user_id: UserId,
+
+  // TODO(server-3.0): New in FL 1, replacing bot_owner
+  bot_owner_id?: number | null,
+
+  // TODO(server-3.0): Replaced in FL 1 by bot_owner_id
+  bot_owner?: string,
+
+  is_active: boolean,
```

Comment:
api types: Add `is_active` to User type
    
    The doc says it is present in `realm_users` and
    `realm_non_active_users`; trust that.

Huh. It seems to actually contradict itself on that:
> realm_non_active_users: (object)[]
>
> Present if realm_user is present in fetch_event_types.
>
> A array of dictionaries where each entry describes a user whose account has been deactivated. Note that unlike the usual User dictionary this does not contain the is_active key as all the users present in this array have deactivated accounts.

even though it then goes on to list `is_active` in the actual list of properties for those objects.

Same for `realm_users`.

Reply:
Hmm, fun. I'll bring it up in `#api documentation`.

Reply:
https://chat.zulip.org/#narrow/stream/412-api-documentation/topic/.60is_active.60.20in.20.60.2Fregister.60.20response/near/1371430